desktop/ui/src/main/java/org/datacleaner/widgets/DCPopupBubble.java

```diff
@@ -34,8 +34,16 @@
 
 public class DCPopupBubble {
 
+    public interface PopupCallback {
```

Comment:
should be public static interface (it's a detail - but so that the interface is statically defined and does not pertain to the instance).